syft/frameworks/torch/nn/rnn.py

```diff
@@ -47,10 +42,12 @@ def init_hidden(self, input):
         This method initializes a hidden state when no hidden state is provided
         in the forward method. It creates a hidden state with zero values.
         """
+        import syft.frameworks.torch.tensors.interpreters.precision as precision
```

Comment:
Doesn't it work with the import at the beginning of the file?

Reply:
No. This is the only configuration (afaik).
Otherwise a major refraction/making it independant of that requirement is required.